recipes/expat/all/conanfile.py

```diff
@@ -11,7 +11,7 @@ class ExpatConan(ConanFile):
     homepage = "https://github.com/libexpat/libexpat"
     license = "MIT"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False], "char_type": ["char", "wchar_t"]}
+    options = {"shared": [True, False], "fPIC": [True, False], "char_type": ["char", "wchar_t", "ushort"]}
```

Comment:
https://github.com/libexpat/libexpat/blob/R_2_4_1/expat/CMakeLists.txt#L140

Reply:
maybe split this in one line per option, it becomes hard to read ;)

Reply:
It proves that you are human!